src/Transformer/CallableTransformer.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AutoMapper\Transformer;
+
+use AutoMapper\Generator\UniqueVariableScope;
+use AutoMapper\Metadata\PropertyMetadata;
+use PhpParser\Node\Arg;
+use PhpParser\Node\Expr;
+use PhpParser\Node\Scalar;
+
+class CallableTransformer implements TransformerInterface
+{
+    public function __construct(private string $callable)
```

Comment:
Is that because we can't handle "real" callable in attributes ? (things like `fn($var) => $var` for exemple ?

Reply:
Yes otherwise it's "impossible" to generate (we can use the previous callback extractor but imo this is a rabbithole)